sdk-python/agentcube/clients/control_plane.py

```diff
@@ -0,0 +1,111 @@
+import os
+import requests
+from typing import Dict, Any, Optional
+
+from agentcube.utils.log import get_logger
+from agentcube.utils.utils import read_token_from_file
+
+class ControlPlaneClient:
```

Comment:
From user's view, i would recommend we merge it into code interpreter. 
They donot care about whether cp or dp